wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -707,6 +707,23 @@ def admin_theme_classname(context):
     contrast_name = contrast_name.split("_")[0]
     return f"w-theme-{theme_name} w-density-{density_name} w-contrast-{contrast_name}"
 
+@register.simple_tag(takes_context=True)
+def admin_theme_user(context):
+    """
+    Retrives theme of current user
+    """
+    user = context["request"].user
+    theme_name = (
+            user.wagtail_userprofile.theme
+            if hasattr(user,"wagtail_userprofile")
+            else "system"
+            )
+    if theme_name == "dark":
+        return "dark light"
+    elif theme_name == "light":
+        return "light dark"
+    else :
+        return "dark light"
```

Comment:
This could probably be simplified, e.g.

```suggestion
    if theme_name == "light":
        return "light dark"
    return "dark light"
```

Reply:
After further research, we should only return a single value if the user has changed the theme setting to an explicit theme. So this should be something like

```suggestion
    if theme_name == "system":
        return "dark light"
    return theme_name
```